tools/export_slow_tests.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+from tools.stats_utils.s3_stat_parser import get_previous_reports_for_branch, Report, Version2Report
+from typing import cast, Dict, Tuple, List
+
+SLOW_TESTS_FILE = '.pytorch-slow-tests'
+SLOW_TEST_CASE_THRESHOLD = 60.0
+
+
+def get_test_case_times() -> Dict[str, float]:
+    # an entry will be like ("test_doc_examples (__main__.TestTypeHints)" -> (current_avg, # values))
+    reports: List[Report] = get_previous_reports_for_branch('origin/viable/strict', "")
+    test_names_to_times: Dict[str, Tuple[float, int]] = dict()
```

Comment:
(see below for context)
```suggestion
    from collections import defaultdict
    from typing import DefaultDict
    test_names_to_times: DefaultDict[str, List[float]] = defaultdict(list)
```

Reply:
+1 for defaultdict

Reply:
ah yes thank you